docs/general/glossary.md

```diff
@@ -521,6 +521,11 @@ will have to use a bridge. Previously known as the
 The previous name for the {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }}
 [Host](#polkadot-host).
 
+## Paseo
+
+The [testnet](#testnet) is set aside for testing the Polkadot Relay Chain. The Paseo testnet is an
+exact copy of Polkadot, and it is maintained by the community.
```

Comment:
It will have exact copy for most of the time, but will deploy fellowship proposed features early providing a brief window of time where parachain devs can benefit from testing on Polkadot "production" grade runtime https://forum.polkadot.network/t/the-new-polkadot-community-testnet/4956

```suggestion
Paseo [testnet](#testnet) provisions testing on Polkadot's "production" runtime, which means less chance of feature/code mismatch when developing parachain apps. Specifically, after the [Polkadot Technical fellowship](#technical-fellowship) proposes a runtime upgrade for Polkadot, this testnet is updated, giving a period where the testnet will be ahead of Polkadot to allow for testing.
```